helpers/handlebars.js

```diff
@@ -212,6 +212,16 @@ hbs.registerHelper("md_partial", function md_partial(partial, opts) {
 	return final;
 });
 
+hbs.registerHelper("get_police_encounter", function log(police) {
+	const startingStr = "As you arrive at the hotel, you see ";
+
+	if (police.officers > 1) {
+		return `${startingStr} ${police.officers} polices arrive behind you. They have ${police.total_hp} HP.`;
```

Comment:
It should say "...police officers arrive behind you", or just "...officers arrive behind you", but not polices.

Reply:
I will change all the `police` to `officer` to standardise stuff.